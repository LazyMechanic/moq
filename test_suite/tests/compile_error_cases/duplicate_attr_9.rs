#[moq::automock]
trait Trait {
    #[moq(default, default)]
    fn f(&self) {}
}

fn main() {}
