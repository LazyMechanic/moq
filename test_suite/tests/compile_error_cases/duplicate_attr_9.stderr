error: duplicate moq attribute: `default`
 --> tests/compile_error_cases/duplicate_attr_9.rs:3:20
  |
3 |     #[moq(default, default)]
  |                    ^^^^^^^
